src/test/java/org/opensearch/security/PitIntegrationTests.java

```diff
@@ -229,10 +229,10 @@ public void testDataStreamWithPits() throws Exception {
         Assert.assertEquals(HttpStatus.SC_OK, resc.getStatusCode());
         String pitId2 =  resc.findValueInJson("pit_id");
 
-        // since pit-2 doesn't have permission to backing data stream indices, throw security error
+        // since pit-3 doesn't have permission to backing data stream indices, throw security error
         resc = rh.executeGetRequest("/_cat/pit_segments",
                 "{\"pit_id\":\"" + pitId2 +"\"}",
-                encodeBasicHeader("pit-2", "nagilum"));
+                encodeBasicHeader("pit-3", "nagilum"));
```

Comment:
`pit-2` permissions got change too? looks like they are having this permission now that they used to not have?